Makefile

```diff
@@ -84,7 +84,7 @@ endif
 .PHONY: $(PLATFORMS)
 
 test:
-	go test -short -p 1 ./...
+	ulimit -n 1024; go test -short -p 1 ./...
```

Comment:
my mac does not allow a ulimit -n which is greater than 400, did you check this locally?